gdal/src/main/scala/geotrellis/gdal/io/hadoop/NetCdfInputFormat.scala

```diff
@@ -1,153 +0,0 @@
-/*
```

Comment:
Why NetCDF format is removed O: ?

Reply:
That's why there is `NetCDFInputFormat`, because GDAL supports it. NetCdfInput format allows to read NetCdf files into Spark memory via gdal netcdf support, it still uses GDALInputFormat but this class covers some specific to netcdf metadata that could be parsed from the netcdf file via GDAL

Reply:
See examples: https://github.com/geotrellis/geotrellis-gdal/blob/dbe65b88c1bfd90fdad582858143a9b6c2a77741/gdal/src/main/scala/geotrellis/gdal/io/hadoop/GdalHadoopSparkContextMethods.scala#L67 

and https://github.com/geotrellis/geotrellis-gdal/blob/dbe65b88c1bfd90fdad582858143a9b6c2a77741/gdal/src/main/scala/geotrellis/gdal/io/hadoop/GdalHadoopSparkContextMethods.scala#L29-L38